lib/Wikia/src/Logger/LogstashFormatter.php

```diff
@@ -20,6 +20,7 @@ public function __construct()
 
 	protected function formatV0(array $record) {
 		$message = array(
+			'appname' => 'mediawiki',
```

Comment:
So right now have `mediawiki` value in `@fields.app_name	` and `program` fields. Any plan to just set it in one place?